platform-api/tests/test_form.py

```
@@ -0,0 +1,118 @@
+        mock.optional_secrets = {
+            "sanity_api_write_token": "test-write-token",
+            "turnstile_secret_key": "test-turnstile-secret",
+            "discord_webhook_url": "http://fake-discord.url"
+        }
+        mock.env_vars = {
+            "sanity_project_id": "test",
+        }
+        mock.kv = mock_kv
+        return mock
+
+    app.dependency_overrides[get_sanity] = lambda: MockSanityClient()
```

Ruff (0.15.12)

[warning] Lambda may be unnecessary; consider inlining inner function

Inline function call

(PLW0108)